### PR TITLE
Fix #1246: Recently Played Highfi Landscape

### LINKED ISSUE #1246: Full UI: Recently Played Activity
This issue is tracking the full UI of RecentlyPlayedActivity

Portrait Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/2b6c8e95-5ac7-4472-bc2b-1c318a3f7e5c/HP-Continue-Playing-/

Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0e6ba726-1930-4f39-9c73-51df10a1c766/HP-Continue-Playing-/

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #1246 

Mock Link: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0e6ba726-1930-4f39-9c73-51df10a1c766/specs/

Note: That the implementation does not follow mocks at two places:
1. The margins between middle and side stories is not as same as mocks because that would lead to UI breakage. You can try that by change value to lower that 36dp.
2. The recyclerview - paddingBottom is `108dp` in code but mocks show it as `160dp` which is incorrect because we are using `160dp` when in portrait mode and `108dp` when in landscape mode.

## Screenshots
Screenshot_1595822491
Screenshot_1595822497
Screenshot_1595822807
Screenshot_1595822825
Screenshot_1595822828
Screenshot_1595822849
Screenshot_1595822852
Screenshot_1595822880

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/section_title.xml [hunk 1]
```diff
@@ -27,9 +27,9 @@
       android:id="@+id/section_title_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="96dp"
+      android:layout_marginStart="72dp"
       android:layout_marginTop="28dp"
```
Comment (MohamedMedhat1998) [inline app/src/main/res/layout-land/section_title.xml:31]: According to the mocks, this should be `32dp`  
Also on line 22 of the same file, `android:layout_marginTop="28dp"` should be `android:layout_marginTop="32dp"`

File: app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt [hunk 2]
```diff
@@ -100,19 +100,19 @@ class OngoingListAdapter(
...
               )
-              storyGridPosition % spanCount == 2 -> params.setMargins(
+              (storyGridPosition) % spanCount == 2 -> params.setMargins(
                 marginMin,
                 marginTop,
                 marginMin,
                 marginBottom
               )
-              storyGridPosition % spanCount == 0 -> params.setMargins(
+              (storyGridPosition) % spanCount == 0 -> params.setMargins(
```
Comment (MohamedMedhat1998) [inline app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt]: Why do we need these braces?

**Review by MohamedMedhat1998 — CHANGES_REQUESTED** [review]
In `land\recently_played_fragment.xml` the RecyclerView top margin needs to be updated to `8dp` instead of `12dp`  
```
<androidx.recyclerview.widget.RecyclerView
        android:id="@+id/ongoing_story_recycler_view"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:clipToPadding="false"
        android:overScrollMode="never"
        android:paddingTop="12dp"
        android:paddingBottom="108dp"
        android:scrollbars="none" />
```
should be  
```
<androidx.recyclerview.widget.RecyclerView
        android:id="@+id/ongoing_story_recycler_view"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:clipToPadding="false"
        android:overScrollMode="never"
        android:paddingTop="8dp" //That is the change
        android:paddingBottom="108dp"
        android:scrollbars="none" />
```
because the `divider_view` in `section_title.xml` will have `32dp` top margins according to the mocks. So the RecyclerView top margin will be `40 - 32 = 8`

File: app/src/main/res/layout-land/section_title.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/section_title.xml:31]: Done.

**Comment by rt4914** [comment]
@MohamedMedhat1998 you can check my last commit which does all the changes mentioned by you.

File: app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt (hunk 2 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/home/recentlyplayed/OngoingListAdapter.kt]: Removed braces.

**Review by MohamedMedhat1998 — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**